File: sdkwork-app-sdk-go/types/vote_vo.go
package types

// 投票响应
type VoteVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	VoteId string `json:"voteId"`
	ContentType string `json:"contentType"`
	ContentId int `json:"contentId"`
	Rating string `json:"rating"`
}

File: sdkwork-app-sdk-csharp/Models/VoteVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VoteVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? VoteId { get; set; }
        public string? ContentType { get; set; }
        public int? ContentId { get; set; }
        public string? Rating { get; set; }
    }
}
